### ISSUE TITLE
Rule "Form controls must have unique ids" isn't implemented properly

### ISSUE THREAD

**Issue by d9su** [globant-ui/arialinter#19]
The formUniqueId rule under inputAssistance.js isn't implemented properly, as its message says "All forms must have unique ids" which isn't consistent with the oaa-accessibility standard. It is also missing the ruleUrl along with a couple other rules.

According to rule url http://oaa-accessibility.org/wcag20/rule/83/ which seems to be the rule that this validation is targeting at, unique ids are required only from buttons, inputs, select, textarea, etc, but not the form itself.

Also, do you think it is necessary to report the whole html of error element's parent? The file name is already included in the error report, maybe it is enough to just report the error element itself. This goes true with other rules, too.
